app/controllers/distributions_controller.rb

```diff
@@ -96,6 +96,12 @@ def new
   def show
     @distribution = Distribution.includes(:line_items).includes(:storage_location).find(params[:id])
     @line_items = @distribution.line_items
+
+    @total_quantity = @line_items.sum(&:quantity)
+    @total_package_count = @line_items.sum { |item| item.has_packages&.ceil || 0 }
```

Comment:
Hmm, I don't know about using `ceil`, if users inputted 1.2 it could be mean it means something to the person who inputted it that way. I would suggest not using `ceil` in this case.